### ISSUE TITLE
sp_Blitz: add check for jobs with DBCC SQLPERF

### ISSUE THREAD

**Issue by BrentOzar** [BrentOzarULTD/SQL-Server-First-Responder-Kit#833]
**Do you want to request a *feature* or report a *bug*?**
Feature

**What is the current behavior?**
Well-meaning folks clear their wait stats by copy/pasting jobs from the Interwebz without realizing that they're hosing monitoring tools.

**What is the expected behavior?**
Use the same logic as the DBCC SHRINK* checks.

**Comment by BlitzErik** [comment]
I found this in a stored procedure that was called from an agent job. Not saying that you shouldn't add this as-is, but it might be worth hitting the default trace to find recent DBCC commands (or the system_health XE, if they're in there).

**Comment by BlitzErik** [comment]
Here's an example that's easy enough to implement:

```
SELECT t.TextData, t.NTUserName, t.NTDomainName, t.HostName,
       t.ApplicationName, t.LoginName, t.StartTime,  t.DBUserName
    FROM sys.traces AS i
    CROSS APPLY sys.fn_trace_gettable(i.path, DEFAULT) AS t
    WHERE i.is_default = 1
	AND CONVERT(NVARCHAR(MAX), t.TextData) IN (
						N'DBCC DROPCLEANBUFFERS',
						N'DBCC FREEPROCCACHE',
						N'DBCC SQLPERF(''sys.dm_os_wait_stats'',CLEAR)'
					)
    ORDER BY t.StartTime DESC
```
On a couple servers, I have these results:

![image](https://cloud.githubusercontent.com/assets/11590415/24864962/d0d5a4bc-1dd3-11e7-9f5f-fbac1fcf4a91.png)

![image](https://cloud.githubusercontent.com/assets/11590415/24865047/08b93fba-1dd4-11e7-8964-6e9a3865f7aa.png)

**Comment by BlitzErik** [comment]
Closed via #857
